dev/PushNotifications/PushNotificationsLongRunningTask.StartupTask/winmain.cpp

```diff
@@ -3,19 +3,20 @@
 
 #include "../PushNotifications-Constants.h"
 
-#include <PushNotificationsLRP_h.h>
+#include <NotificationsLongRunningProcess_h.h>
 #include <wil/resource.h>
 
 int WINAPI WinMain(HINSTANCE /*hInstance*/, HINSTANCE /*hPrevInstance*/, PSTR /*lpCmdLine*/, int /*nCmdShow*/)
 {
+    Sleep(20000);
+
     RETURN_IF_FAILED(::CoInitializeEx(nullptr, COINITBASE_MULTITHREADED));
```

Comment:
RETURN_IF_FAILED is the wrong macro to us. This method should be returning win32 errors right?

Reply:
Please look into best practises for returning error codes in WinAPI's

Reply:
Will this be addressed?

Reply:
DynamicDependency COM server does exactly the same thing. Many WinMains in Windows codebase return HRESULTS too.